code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -9,8 +9,6 @@
  * Glass sheets
  */
 GLOBAL_LIST_INIT(glass_recipes, list ( \
-	new/datum/stack_recipe("directional window", /obj/structure/window/unanchored, time = 0, on_floor = TRUE, window_checks = TRUE), \
```

Comment:
Not needed. All abnormalities can damage objects.

Reply:
Scorched girl can't.